src/libsyntax/parse/mod.rs

```diff
@@ -33,7 +33,6 @@ pub mod attr;
 
 pub mod common;
 pub mod classify;
-pub mod obsolete;
```

Comment:
I was thinking about it... Do you think this module will be useful after reaching post 1.0, when no backs-incompatible changes are allowed?